oem/firstvoices/windows/src/inst/Makefile

```diff
@@ -11,7 +11,7 @@ MSI=firstvoices.msi
 EXE=firstvoices.exe
 KMP=fv_all.kmp
 INTEXE=firstvoices-fv_all.exe
-APPTITLE="FirstVoices Keyboards"
+APPTITLE="Keyman for FirstVoices"
```

Comment:
This name was not used in many places; the main screen showed "Keyman Desktop for FirstVoices" in both the logo and the text. So this was technically somewhat inconsistent.

I've stuck with the "Keyman for FirstVoices" name for now unless they tell us otherwise.